src/utils/whitelist-tokens.ts

```diff
@@ -1,3 +1,14 @@
+export type WhitelistedToken = {
```

Comment:
nit: naming conventions
```suggestion
export type AllowlistedToken = {
```